content/cloud/serverless-ai.md

```diff
@@ -0,0 +1,65 @@
+title = "Serverless AI"
+template = "cloud_main"
+date = "2023-09-05T09:00:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/serverless-ai.md"
+
+---
+- [Accessing Private Beta](#accessing-private-beta)
+- [Quotas And Service Limitations For Fermyon Serverless AI](#quotas-and-service-limitations-for-fermyon-serverless-ai)
+- [FAQ](#faq)
+- [Next Steps](#next-steps)
+
+## Accessing Private Beta
+
+The Fermyon Serverless AI is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/mNzgXRvB#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space, and all requests cannot be guaranteed. 
+
+Once you have access to the private beta, please ensure you have:
+- TODO (what does a user need to install/upgrade) i.e. [Spin CLI](./cli-reference.md) version TODO or greater
+- TODO (what does a user need to install/upgrade) i.e. [TODO plugin](TODO) version TODO or greater installed. 
+
+## Quotas And Service Limitations For Fermyon Serverless AI
+
+*Quotas* 
+* You can have up to 75 inferencing requests per hour, 200 embedding requests per hour
+* You can have a maximum of 1,024 tokens (prompt and response) per request
+
+>> If you have questions about increasing these limits, please reach out to [support@fermyon.com](mailto://support@fermyon.com)
+
+*Service Limitations*
+* TODO any service limitations to run Fermyon Serverless AI (i.e. only supports TODO).
```

Comment:
We currently on support Code Llama, Llama 2, and all-MiniLM-L6-v2

Reply:
We might want to be a bit more specific with the llama models as there are other flavors:
* code llama instruct 
* llama2 chat